libgit2-sys/lib.rs

```diff
@@ -79,6 +79,16 @@ impl Clone for git_strarray {
 }
 
 #[repr(C)]
+#[derive(Copy)]
+pub struct git_oidarray {
+    pub ids: *mut git_oid,
+    pub count: size_t,
+}
+impl Clone for git_oidarray {
+    fn clone(&self) -> git_oidarray { *self }
+}
```

Comment:
either Copy or Clone seems to be required for the Binding::raw method on OidArray
the Clone methods is implemented in terms of Copy.
seems like it'd be easier to leave as is.